src/lib/webengine/webview.cpp

```diff
@@ -522,7 +522,7 @@ void WebView::openUrlInNewTab(const QUrl &url, Qz::NewTabPositionFlags position)
 void WebView::openActionUrl()
 {
     if (QAction* action = qobject_cast<QAction*>(sender())) {
-        load(action->data().toUrl());
+        openUrlInNewTab(action->data().toUrl(), Qz::NT_SelectedTab);
```

Comment:
This has nothing to do with "go to web address" action, you just changed the behavior of `openActionUrl` method that is used for more action that just go to web address.

As you can see on line 871, you can middle-click the action to open in new tab.